android/assets/jsons/translations/template.properties

```diff
@@ -1254,8 +1254,13 @@ Granted by: =
 [bonus] with [tech] = 
 Difficulty levels = 
 The possible rewards are: = 
+Eras = 
+Embarked strength: [amount]† = 
+Base unit buy cost: [amount]¤ = 
+Research agreement cost: [amount]¤ = 
 
-# Policies
+
+S# Policies
```

Comment:
S?

Reply:
Oops, shrapnel typo. Good thing it's neutral - `#` is not the thing marking a comment line in translations, lack of `=` is.